openlibrary/plugins/inside/code.py

```diff
@@ -2,16 +2,15 @@
 from time import time
 import re
 import simplejson
-import httplib
 import logging
 
 import web
 from infogami.utils import delegate
 from infogami.utils.view import render_template
 from openlibrary.core.fulltext import fulltext_search
 
-
 from six.moves import urllib
+import six.moves.http_client as httplib
```

Comment:
Neither `urllib` nor `httplib` appear to be used in this file -- are they safe to remove?